discovery-infra/tests/base_test.py

```diff
@@ -77,3 +82,10 @@ def assert_cluster_validation(cluster_info, validation_section, validation_id, e
     def assert_string_length(string, expected_len):
         assert len(string) == expected_len, "Expected len string of: " + str(expected_len) + \
                                             " rather than: " + str(len(string)) + " String value: " + string
+
+    @staticmethod
+    def collect_test_logs(cluster, api_client, test_name):
+        log_dir_name = f'{consts.LOG_FOLDER}/{test_name}'
+        with suppress(ApiException):
```

Comment:
I think there is no need to suppress ApiException while `download_logs` should do it

Reply:
In some tests the cluster is deleted during the test e.g https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/tests/test_general.py#L17
suppress ApiException addresses this